index.html

```diff
@@ -44,129 +50,172 @@ <h1>I'm Abigael <br> Glad to see <br> you</h1>
 
     <!-- Work section  -->
     <div class="work-section">
+        <!-- card1 -->
         <section class="card work">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_103](https://user-images.githubusercontent.com/69638013/166301019-65e17989-c6ba-4b94-9c65-dc5bbcd2f91f.png)  |  ![Screenshot_101](https://user-images.githubusercontent.com/69638013/166300985-5bd21b30-9a65-4c1f-aa48-20ae5f3f281d.png)

- [ ] Kindly remove the white space behind each card from  768px, at 1024px and 14440px this white space doesn't occur 👍🏿 

### OPTIONAL 


- [ ] All cards in this section should be wrapped in one section, this will make your project more semantic 👍🏿

Reply:
That white space was actually because of a margin. That has been corrected